packages/@aws-cdk/cloud-assembly-schema/schema/assets.schema.json

```diff
@@ -162,6 +162,13 @@
                 "platform": {
                     "description": "Platform to build for. _Requires Docker Buildx_.\n\nSpecify this property to build images on a specific platform/architecture. (Default - current machine platform)",
                     "type": "string"
+                },
+                "dockerOutputs": {
+                    "description": "Outputs (Default - No outputs)",
```

Comment:
Needs better description (I know these are repetitive)

Reply:
FIY: This file might actually be generated....

Reply:
Yep, it is. Thanks!